app/controllers/idv/in_person/state_id_controller.rb

```diff
@@ -148,11 +148,24 @@ def parsed_dob
 
       def pii
         data = pii_from_user
-        data = data.merge(flow_params) if params.has_key?(:state_id)
+        if params.has_key?(:identity_doc) || params.has_key?(:state_id)
+          data = data.merge(flow_params)
+        end
         data.deep_symbolize_keys
       end
 
       def flow_params
+        if params.dig(:identity_doc).present?
```

Comment:
Rather than re-mapping the field name, can we update the code to use the new field name?

Might require some [temporary juggling to manage 50/50 state](https://handbook.login.gov/articles/manage-50-50-state.html), but I think it'd be more maintainable long-term to just have a single set of parameter names, than continue to deal with mapping parameter names.

Reply:
Good question and suggestion. I originally went down this path and concluded that renaming the data would result in less consistency overall, for example the areas that use data coming from this particular form would now be called id_number, whereas some other areas in the code would still use the name state_id_number.

This issue this change addresses is caused by the form name only - the more I thought about it the more it seemed unideal to change the param names just for the sake of a workaround, especially if it introduces some inconsistencies.

As for continuing to deal with mapping parameter names - this is a workaround unique to the state id form, I don't think we'd have to worry about renaming params anywhere else.

Reply:
I went ahead and renamed the state_id_number fields to id_number to reflect the field name! Turns out the rest of the data is unaffected by the new form name, since the controller parses the params out.

Reply:
Update, after more thought and discussion I'm resetting back to the original solution that translates the names. There is shared code between the IPP/non-IPP flows that use the changed field names, which adds complexity that isn't worth it for this ticket.